hvantk/visualization/qc_report.py

```
@@ -0,0 +1,736 @@
+            <div class="alert alert-success">
+                <strong>✅ Overall Quality:</strong> No major quality issues detected. 
+                Dataset appears to be of good quality for downstream analysis.
+            </div>
+        """)
+
+    return ''.join(recommendations)
+
+
+def generate_qc_report(qc_results,
+                      output_path: Union[str, Path],
+                      title: str = "Quality Control Report",
```

Ruff (0.14.3)

Unused function argument: `title`

(ARG001)